public/pages/PolicyDetails/components/PolicySettings/PolicySettings.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React, { Component } from "react";
+import { EuiFlexGrid, EuiSpacer, EuiFlexItem, EuiText, EuiBasicTable } from "@elastic/eui";
+import { ContentPanel, ContentPanelActions } from "../../../../components/ContentPanel";
+import { ModalConsumer } from "../../../../components/Modal";
+import { ISMTemplate } from "../../../../../models/interfaces";
+
+interface PolicySettingsProps {
+  policyId: string;
+  channelId: string;
+  primaryTerm: number;
+  lastUpdated: string;
+  description: string;
+  sequenceNumber: number;
+  schemaVersion: number;
+  ismTemplates: ISMTemplate[];
+}
+
+interface PolicySettingsState {
+  pageIndex: number;
+  pageSize: number;
+  showPerPageOptions: boolean;
+}
+
+export default class PolicySettings extends Component<PolicySettingsProps, PolicySettingsState> {
+  constructor(props: PolicySettingsProps) {
+    super(props);
+
+    this.state = {
+      pageIndex: 0,
+      pageSize: 10,
+      showPerPageOptions: true,
+    }
+  }
+
+  // TODO: Needs to be updated to handle template array
```

Comment:
Is this TODO still true? I see it's handling ISMTemplates as an array

Reply:
Nope, removing TODO.